NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(orderly_artefact)
+export(orderly_depends)
```

Comment:
I guess the most consistent with the other function names would be `orderly_dependency`